pkg/convert/pot.go

```diff
@@ -0,0 +1,70 @@
+package convert
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/pot"
+)
+
+func ToPot(m model.Messages) ([]byte, error) {
+	var b bytes.Buffer
+
+	b.WriteString(fmt.Sprintf("\"Language: %s\n", m.Language))
+
+	for _, message := range m.Messages {
+		if message.Description != "" {
+			b.WriteString(fmt.Sprintf("#. %s\n", message.Description))
+		}
+
+		if message.Fuzzy {
+			b.WriteString("#, fuzzy\n")
+		}
+
+		b.WriteString(fmt.Sprintf("msgid \"%s\"\n", message.ID))
+
+		b.WriteString(fmt.Sprintf("msgstr \"%s\"\n", message.Message))
+	}
+
+	return b.Bytes(), nil
+}
+
+func FromPot(b []byte) (model.Messages, error) {
+	var po *pot.Po
+	if err := json.Unmarshal(b, &po); err != nil {
+		return model.Messages{}, fmt.Errorf("error unmarshaling byte slice to pot.Po: %w", err)
+	}
+
+	messages := make([]model.Message, 0, len(po.Messages))
+
+	for _, node := range po.Messages {
+		if node.MsgIdPlural != "" {
```

Comment:
I am curious why are we skipping plurals ?

Reply:
Because in our model.Messages structure we don't support plurals. I had conversation with @janishorsts about this ;)

Reply:
https://github.com/orgs/translate-agent/projects/1/views/1?pane=issue&itemId=21251425